source/docs/steeltoe-circuitbreaker.html.markdown

```diff
@@ -1166,10 +1263,12 @@ public class Program {
     ...
 ```
 
-When pushing the application to Cloud Foundry, the settings from service bindings will merge with the settings from other configuration mechanisms (e.g. `appsettings.json`).
+Both sources are then added to the configuration builder.
+
+When pushing the application to Cloud Foundry, the settings from service bindings merge with the settings from other configuration mechanisms (such as `appsettings.json`).
 
-If there are merge conflicts, the last provider added to the Configuration will take precedence and override all others.
+If there are merge conflicts, the last provider added to the Configuration take precedences and overrides all others.
```

Comment:
I don't believe you can pluralize "precedence".  Should it be "takes precedence"

Reply:
I may be wrong, but it sounded off.